tests/test_helper.bash

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if [ -z "$STACKER_NAMESPACE" ]; then
```

Comment:
maybe make this a random string if not specified?  it's pretty easy to generate one